packages/@aws-cdk-testing/framework-integ/test/aws-ecs-patterns/test/sqs-reader/index.py

```diff
@@ -6,17 +6,27 @@
 print('QUEUE_NAME ' + QUEUE_NAME)
 
 if __name__ == '__main__':
-    client = boto3.client('sqs')
-    queue_url = client.get_queue_url(QueueName=QUEUE_NAME)['QueueUrl']
-    print('queue_url ' + queue_url)
-    while True:
-        response = client.receive_message(
-            QueueUrl=queue_url,
-            WaitTimeSeconds=10,
-        )
-        if response and 'Messages' in response:
-            for msg in response['Messages']:
-                print(msg['Body'])
-            entries = [{'Id': x['MessageId'], 'ReceiptHandle': x['ReceiptHandle']} for x in response['Messages']]
-            client.delete_message_batch(QueueUrl=queue_url, Entries=entries)
+    try:
+        client = boto3.client('sqs', region_name='ap-southeast-2')
```

Comment:
```suggestion
        client = boto3.client('sqs')
```
Let's leave the default region on the final file.

Reply:
Removed, thanks.